src/status_im/utils/keychain/core.cljs

```diff
@@ -1,10 +1,12 @@
 (ns status-im.utils.keychain.core
   (:require [re-frame.core :as re-frame]
             [taoensso.timbre :as log]
-            [status-im.react-native.js-dependencies :as rn]))
+            [status-im.react-native.js-dependencies :as rn]
+            [status-im.utils.platform :as platform]))
 
 (def key-bytes 64)
 (def username "status-im.encryptionkey")
+(when platform/desktop? (.setUsername rn/keychain username))
```

Comment:
Using a side-effecful expression at ns top-level feels a bit odd (if it's not a `def`). Maybe it makes sense to move it to one of init fns somewhere?